### PR TITLE
Add scripts-to-rule-them-all

### PR DESCRIPTION
**Pull request by anneschuth**
This adds `script/test` and `script/bootstrap`, with code as described in #1  (adding config for running this on CI is left for a separate PR).

### CODE REVIEW

File: script/test [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Install cld2-cffi package with an additional CFLAG
+CFLAGS="-Wno-narrowing" pip3 install --quiet cld2-cffi
+
+# Install requirements
+pip3 install --quiet --requirement requirements.txt
+
+# Install and run pylint
+pip3 install --quiet --upgrade pylint 
+python3 -m pylint -f parseable textpipe | tee pylint.out
```
Comment (anneschuth) [inline script/test]: Maybe consider flake8?

Comment (TanjaCrijns) [reply script/test]: In my experience, pylint is a more detailed than flake8 even though the output may be a bit much at times (but you can easily disable the warnings or conventions). What are your thought on flake8 vs pylint?

Comment (dodijk) [reply script/test]: Also only a slight preference from our side. We like that it actually assigns a score to the code that we can track over time and with PRs. I would propose to keep using `pylint` for now.

**Comment by dodijk** [comment]
Maybe we can consider not having a `script/test` file, as we are using very standard Python tools. I see many packages that have Travis integration without a `script/test`. We could start with specifing the details in our `.travis.yml` and describe them in the `README`. How do you feel about this, @anneschuth?

**Comment by anneschuth** [comment]
I'm still a big fan of https://github.com/github/scripts-to-rule-them-all. I'd prefer to have that script/test so that I can run the same tests locally and on travis.

**Comment by dodijk** [comment]
OK, I can see how that makes this a bit cleaner and our `.travis.yml` less complex.

File: script/test [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Install cld2-cffi package with an additional CFLAG
+CFLAGS="-Wno-narrowing" pip3 install --quiet cld2-cffi
```
Comment (dodijk) [inline script/test]: Not sure if this is needed on Travis. I needed it on our Jenkins machine.

File: script/test [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+# Install requirements and test requirements
+pip3 install --quiet --requirement requirements.txt
```
Comment (dodijk) [inline script/test]: Move this to `script/bootstrap`? And drop the explicit references to `pip3` and `python3` everywhere?

**Pull request merged**